### PR TITLE
pull up to kvfilter 0.7 which offers optional Regex filtering as well 

### PR DESCRIPTION
**Pull request by przygienda**

### CODE REVIEW

File: src/file.rs [hunk 1]
```diff
@@ -85,11 +86,15 @@ impl FileLoggerBuilder {
         level: Severity,
```
Comment (sile) [inline src/file.rs]: I prefer to use a builder or option struct instead of many arguments.
(e.g., Export `KVFilterParameters` and use it as the argument)

Comment (przygienda) [reply src/file.rs]: ok ...

Comment (przygienda) [reply src/file.rs]: done

**Comment by przygienda** [comment]
should be ready as you requested ...

**Comment by sile** [comment]
Thanks!

**Pull request merged**